src/ssl/ensemble.py

```diff
@@ -53,14 +69,15 @@ def drop_ensemble(self):
         """
         self.ensemble = []
 
-    def fit_ensembĺe(self, instances, classes):
+    def fit_ensemble(self, instances, classes):
```

Comment:
```suggestion
    def fit_ensemble(self, instances, classes) -> None:
```
tipos de instance e classes

Reply:
feito